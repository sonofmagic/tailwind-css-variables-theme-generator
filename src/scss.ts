import type { Value, SassColor, SassString, Options } from 'sass'
import type { OrderedMap } from 'immutable'
import * as sass from 'sass'
import defu from 'defu'
// The asynchronous variants are much slower
export function exposeScssVariable (
  exposeFilePath: string,
  options?: Options<'sync'>
) {
  const exposeAarry: OrderedMap<SassString, SassColor>[] = []
  const defaultOption: Options<'sync'> = {
    functions: {
      'expose($map)': (args: Value[]) => {
        const value = args[0].assertMap('map')
        const map = value.contents as OrderedMap<SassString, SassColor>
        exposeAarry.push(map)
        return value
      }
    }
  }
  sass.compile(exposeFilePath, defu(options!, defaultOption))
  return exposeAarry
}
