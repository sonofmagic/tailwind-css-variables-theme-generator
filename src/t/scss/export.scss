@use '{{filepath}}' as C;
@use './util.scss' as Util;

:export {
  @each $var, $color in C.{{variableName}} {
    #{$var}: Util.getRgbString($color);
  }
}
