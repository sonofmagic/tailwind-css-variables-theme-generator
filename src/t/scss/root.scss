@use '{{filepath}}' as C;
@use './util.scss' as Util;

:root {
  @each $var, $color in C.{{variableName}} {
    #{$var}: Util.getRgbString($color);
  }
}