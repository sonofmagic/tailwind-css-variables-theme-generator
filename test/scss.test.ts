import { exposeScssVariable } from '@/scss'
import { resolve } from 'path'
describe('scss expose', () => {
  it('default expose', async () => {
    const exposeAarry = await exposeScssVariable(
      resolve(__dirname, './fixtures/expose/expose.scss')
    )
    const mergedMap: Record<string, string> = {}
    for (let i = 0; i < exposeAarry.length; i++) {
      const map = exposeAarry[i]
      for (const [key, color] of map) {
        mergedMap[key.text] = `${color.red ?? 0} ${color.green ?? 0} ${
          color.blue ?? 0
        }` // ${color.alpha}
      }
    }
    expect(mergedMap).toMatchSnapshot()
  })
})
