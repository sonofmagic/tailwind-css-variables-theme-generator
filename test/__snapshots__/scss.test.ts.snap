// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scss expose default expose 1`] = `
{
  "--color-accent-emphasis": "49 109 202",
  "--color-accent-fg": "83 155 245",
  "--color-accent-muted": "65 132 228",
  "--color-accent-subtle": "65 132 228",
  "--color-action-list-item-danger-active-bg": "229 83 75",
  "--color-action-list-item-danger-hover-bg": "229 83 75",
  "--color-action-list-item-danger-hover-text": "244 112 103",
  "--color-action-list-item-default-active-bg": "144 157 171",
  "--color-action-list-item-default-hover-bg": "144 157 171",
  "--color-action-list-item-default-selected-bg": "144 157 171",
  "--color-action-list-item-inline-divider": "68 76 86",
  "--color-ansi-black": "84 93 104",
  "--color-ansi-black-bright": "99 110 123",
  "--color-ansi-blue": "83 155 245",
  "--color-ansi-blue-bright": "108 182 255",
  "--color-ansi-cyan": "57 197 207",
  "--color-ansi-cyan-bright": "86 212 221",
  "--color-ansi-gray": "99 110 123",
  "--color-ansi-green": "87 171 90",
  "--color-ansi-green-bright": "107 196 109",
  "--color-ansi-magenta": "176 131 240",
  "--color-ansi-magenta-bright": "220 189 251",
  "--color-ansi-red": "244 112 103",
  "--color-ansi-red-bright": "255 147 138",
  "--color-ansi-white": "144 157 171",
  "--color-ansi-white-bright": "205 217 229",
  "--color-ansi-yellow": "198 144 38",
  "--color-ansi-yellow-bright": "218 170 63",
  "--color-attention-emphasis": "150 102 0",
  "--color-attention-fg": "198 144 38",
  "--color-attention-muted": "174 124 20",
  "--color-attention-subtle": "174 124 20",
  "--color-avatar-bg": "205 217 229",
  "--color-avatar-border": "205 217 229",
  "--color-avatar-child-shadow": "0 0 0",
  "--color-avatar-stack-fade": "68 76 86",
  "--color-avatar-stack-fade-more": "55 62 71",
  "--color-border-default": "68 76 86",
  "--color-border-muted": "55 62 71",
  "--color-border-subtle": "205 217 229",
  "--color-btn-active-bg": "61 68 77",
  "--color-btn-active-border": "99 110 123",
  "--color-btn-bg": "55 62 71",
  "--color-btn-border": "205 217 229",
  "--color-btn-counter-bg": "68 76 86",
  "--color-btn-danger-counter-bg": "201 60 55",
  "--color-btn-danger-disabled-bg": "34 39 46",
  "--color-btn-danger-disabled-counter-bg": "201 60 55",
  "--color-btn-danger-disabled-text": "229 83 75",
  "--color-btn-danger-focus-border": "229 83 75",
  "--color-btn-danger-focus-shadow": "0 0 0",
  "--color-btn-danger-hover-bg": "201 60 55",
  "--color-btn-danger-hover-border": "229 83 75",
  "--color-btn-danger-hover-counter-bg": "255 255 255",
  "--color-btn-danger-hover-icon": "205 217 229",
  "--color-btn-danger-hover-inset-shadow": "0 0 0",
  "--color-btn-danger-hover-shadow": "0 0 0",
  "--color-btn-danger-hover-text": "205 217 229",
  "--color-btn-danger-icon": "229 83 75",
  "--color-btn-danger-selected-bg": "173 46 44",
  "--color-btn-danger-selected-border": "244 112 103",
  "--color-btn-danger-selected-shadow": "0 0 0",
  "--color-btn-danger-selected-text": "255 255 255",
  "--color-btn-danger-text": "229 83 75",
  "--color-btn-focus-bg": "55 62 71",
  "--color-btn-focus-border": "118 131 144",
  "--color-btn-focus-shadow": "0 0 0",
  "--color-btn-hover-bg": "68 76 86",
  "--color-btn-hover-border": "118 131 144",
  "--color-btn-inset-shadow": "0 0 0",
  "--color-btn-outline-counter-bg": "49 109 202",
  "--color-btn-outline-disabled-bg": "34 39 46",
  "--color-btn-outline-disabled-counter-bg": "49 109 202",
  "--color-btn-outline-disabled-text": "83 155 245",
  "--color-btn-outline-focus-border": "205 217 229",
  "--color-btn-outline-focus-shadow": "0 0 0",
  "--color-btn-outline-hover-bg": "68 76 86",
  "--color-btn-outline-hover-border": "205 217 229",
  "--color-btn-outline-hover-counter-bg": "205 217 229",
  "--color-btn-outline-hover-inset-shadow": "0 0 0",
  "--color-btn-outline-hover-shadow": "0 0 0",
  "--color-btn-outline-hover-text": "83 155 245",
  "--color-btn-outline-selected-bg": "27 75 145",
  "--color-btn-outline-selected-border": "205 217 229",
  "--color-btn-outline-selected-shadow": "0 0 0",
  "--color-btn-outline-selected-text": "205 217 229",
  "--color-btn-outline-text": "83 155 245",
  "--color-btn-primary-bg": "52 125 57",
  "--color-btn-primary-border": "205 217 229",
  "--color-btn-primary-counter-bg": "205 217 229",
  "--color-btn-primary-disabled-bg": "52 125 57",
  "--color-btn-primary-disabled-border": "205 217 229",
  "--color-btn-primary-disabled-text": "205 217 229",
  "--color-btn-primary-focus-bg": "52 125 57",
  "--color-btn-primary-focus-border": "205 217 229",
  "--color-btn-primary-focus-shadow": "0 0 0",
  "--color-btn-primary-hover-bg": "70 149 74",
  "--color-btn-primary-hover-border": "205 217 229",
  "--color-btn-primary-icon": "205 217 229",
  "--color-btn-primary-inset-shadow": "0 0 0",
  "--color-btn-primary-selected-bg": "52 125 57",
  "--color-btn-primary-selected-shadow": "0 0 0",
  "--color-btn-primary-shadow": "0 0 0",
  "--color-btn-primary-text": "255 255 255",
  "--color-btn-selected-bg": "45 51 59",
  "--color-btn-shadow": "0 0 0",
  "--color-btn-shadow-active": "0 0 0",
  "--color-btn-shadow-input-focus": "0 0 0",
  "--color-btn-text": "173 186 199",
  "--color-canvas-default": "34 39 46",
  "--color-canvas-default-transparent": "32 54 85",
  "--color-canvas-inset": "28 33 40",
  "--color-canvas-overlay": "45 51 59",
  "--color-canvas-subtle": "45 51 59",
  "--color-checks-ansi-black": "34 39 46",
  "--color-checks-ansi-black-bright": "45 51 59",
  "--color-checks-ansi-blue": "83 155 245",
  "--color-checks-ansi-blue-bright": "108 182 255",
  "--color-checks-ansi-cyan": "118 227 234",
  "--color-checks-ansi-cyan-bright": "179 240 255",
  "--color-checks-ansi-gray": "99 110 123",
  "--color-checks-ansi-green": "87 171 90",
  "--color-checks-ansi-green-bright": "107 196 109",
  "--color-checks-ansi-magenta": "176 131 240",
  "--color-checks-ansi-magenta-bright": "220 189 251",
  "--color-checks-ansi-red": "244 112 103",
  "--color-checks-ansi-red-bright": "255 147 138",
  "--color-checks-ansi-white": "144 157 171",
  "--color-checks-ansi-white-bright": "144 157 171",
  "--color-checks-ansi-yellow": "198 144 38",
  "--color-checks-ansi-yellow-bright": "218 170 63",
  "--color-checks-bg": "28 33 40",
  "--color-checks-btn-hover-bg": "99 110 123",
  "--color-checks-btn-hover-icon": "173 186 199",
  "--color-checks-btn-icon": "118 131 144",
  "--color-checks-container-border-width": "0 0 0",
  "--color-checks-donut-error": "229 83 75",
  "--color-checks-donut-neutral": "118 131 144",
  "--color-checks-donut-pending": "198 144 38",
  "--color-checks-donut-success": "70 149 74",
  "--color-checks-dropdown-bg": "45 51 59",
  "--color-checks-dropdown-border": "68 76 86",
  "--color-checks-dropdown-btn-hover-bg": "99 110 123",
  "--color-checks-dropdown-btn-hover-text": "173 186 199",
  "--color-checks-dropdown-hover-bg": "99 110 123",
  "--color-checks-dropdown-hover-text": "173 186 199",
  "--color-checks-dropdown-shadow": "28 33 40",
  "--color-checks-dropdown-text": "173 186 199",
  "--color-checks-gate-bg": "174 124 20",
  "--color-checks-gate-text": "118 131 144",
  "--color-checks-gate-waiting-text": "198 144 38",
  "--color-checks-header-border": "55 62 71",
  "--color-checks-header-icon": "118 131 144",
  "--color-checks-header-label-open-text": "173 186 199",
  "--color-checks-header-label-text": "118 131 144",
  "--color-checks-input-bg": "45 51 59",
  "--color-checks-input-focus-text": "173 186 199",
  "--color-checks-input-placeholder-text": "84 93 104",
  "--color-checks-input-text": "118 131 144",
  "--color-checks-line-dt-fm-bg": "150 102 0",
  "--color-checks-line-dt-fm-text": "205 217 229",
  "--color-checks-line-hover-bg": "99 110 123",
  "--color-checks-line-num-text": "84 93 104",
  "--color-checks-line-selected-bg": "65 132 228",
  "--color-checks-line-selected-num-text": "83 155 245",
  "--color-checks-line-text": "118 131 144",
  "--color-checks-line-timestamp-text": "84 93 104",
  "--color-checks-logline-command-text": "83 155 245",
  "--color-checks-logline-debug-text": "152 110 226",
  "--color-checks-logline-error-bg": "229 83 75",
  "--color-checks-logline-error-num-text": "84 93 104",
  "--color-checks-logline-error-text": "118 131 144",
  "--color-checks-logline-num-text": "84 93 104",
  "--color-checks-logline-section-text": "87 171 90",
  "--color-checks-logline-text": "118 131 144",
  "--color-checks-logline-warning-bg": "174 124 20",
  "--color-checks-logline-warning-num-text": "198 144 38",
  "--color-checks-logline-warning-text": "118 131 144",
  "--color-checks-run-border-width": "0 0 0",
  "--color-checks-scrollbar-thumb-bg": "99 110 123",
  "--color-checks-step-error-text": "229 83 75",
  "--color-checks-step-header-open-bg": "45 51 59",
  "--color-checks-step-warning-text": "198 144 38",
  "--color-checks-text-link": "83 155 245",
  "--color-checks-text-primary": "173 186 199",
  "--color-checks-text-secondary": "118 131 144",
  "--color-codemirror-activeline-bg": "99 110 123",
  "--color-codemirror-bg": "34 39 46",
  "--color-codemirror-cursor": "173 186 199",
  "--color-codemirror-guttermarker-subtle-text": "84 93 104",
  "--color-codemirror-guttermarker-text": "34 39 46",
  "--color-codemirror-gutters-bg": "34 39 46",
  "--color-codemirror-linenumber-text": "118 131 144",
  "--color-codemirror-lines-bg": "34 39 46",
  "--color-codemirror-matchingbracket-text": "173 186 199",
  "--color-codemirror-selection-bg": "65 132 228",
  "--color-codemirror-syntax-comment": "118 131 144",
  "--color-codemirror-syntax-constant": "108 182 255",
  "--color-codemirror-syntax-entity": "220 189 251",
  "--color-codemirror-syntax-keyword": "244 112 103",
  "--color-codemirror-syntax-storage": "244 112 103",
  "--color-codemirror-syntax-string": "150 208 255",
  "--color-codemirror-syntax-support": "108 182 255",
  "--color-codemirror-syntax-variable": "246 157 80",
  "--color-codemirror-text": "173 186 199",
  "--color-counter-border": "0 0 0",
  "--color-custom-text-color": "84 104 33",
  "--color-danger-emphasis": "201 60 55",
  "--color-danger-fg": "229 83 75",
  "--color-danger-muted": "229 83 75",
  "--color-danger-subtle": "229 83 75",
  "--color-diff-blob-addition-fg": "173 186 199",
  "--color-diff-blob-addition-line-bg": "70 149 74",
  "--color-diff-blob-addition-num-bg": "87 171 90",
  "--color-diff-blob-addition-num-text": "173 186 199",
  "--color-diff-blob-addition-word-bg": "70 149 74",
  "--color-diff-blob-deletion-fg": "173 186 199",
  "--color-diff-blob-deletion-line-bg": "229 83 75",
  "--color-diff-blob-deletion-num-bg": "229 83 75",
  "--color-diff-blob-deletion-num-text": "173 186 199",
  "--color-diff-blob-deletion-word-bg": "229 83 75",
  "--color-diff-blob-expander-icon": "118 131 144",
  "--color-diff-blob-hunk-num-bg": "65 132 228",
  "--color-diff-blob-selected-line-highlight-mix-blend-mode": "0 0 0",
  "--color-diffstat-addition-bg": "87 171 90",
  "--color-diffstat-addition-border": "205 217 229",
  "--color-diffstat-deletion-border": "205 217 229",
  "--color-done-emphasis": "130 86 208",
  "--color-done-fg": "152 110 226",
  "--color-done-muted": "152 110 226",
  "--color-done-subtle": "152 110 226",
  "--color-fg-default": "173 186 199",
  "--color-fg-muted": "118 131 144",
  "--color-fg-on-emphasis": "205 217 229",
  "--color-fg-subtle": "84 93 104",
  "--color-header-bg": "45 51 59",
  "--color-header-divider": "118 131 144",
  "--color-header-logo": "205 217 229",
  "--color-header-search-bg": "34 39 46",
  "--color-header-search-border": "68 76 86",
  "--color-header-text": "205 217 229",
  "--color-input-disabled-bg": "99 110 123",
  "--color-marketing-icon-primary": "5 60 116",
  "--color-marketing-icon-secondary": "49 109 202",
  "--color-menu-bg-active": "45 51 59",
  "--color-mktg-btn-bg": "246 248 250",
  "--color-mktg-btn-shadow-focus": "0 0 0",
  "--color-mktg-btn-shadow-hover": "0 0 0",
  "--color-mktg-btn-shadow-hover-muted": "0 0 0",
  "--color-mktg-btn-shadow-outline": "0 0 0",
  "--color-neutral-emphasis": "99 110 123",
  "--color-neutral-emphasis-plus": "99 110 123",
  "--color-neutral-muted": "99 110 123",
  "--color-neutral-subtle": "99 110 123",
  "--color-overlay-shadow": "0 0 0",
  "--color-prettylights-syntax-brackethighlighter-angle": "118 131 144",
  "--color-prettylights-syntax-brackethighlighter-unmatched": "229 83 75",
  "--color-prettylights-syntax-carriage-return-bg": "173 46 44",
  "--color-prettylights-syntax-carriage-return-text": "205 217 229",
  "--color-prettylights-syntax-comment": "118 131 144",
  "--color-prettylights-syntax-constant": "108 182 255",
  "--color-prettylights-syntax-constant-other-reference-link": "150 208 255",
  "--color-prettylights-syntax-entity": "220 189 251",
  "--color-prettylights-syntax-entity-tag": "141 219 140",
  "--color-prettylights-syntax-invalid-illegal-bg": "146 35 35",
  "--color-prettylights-syntax-invalid-illegal-text": "205 217 229",
  "--color-prettylights-syntax-keyword": "244 112 103",
  "--color-prettylights-syntax-markup-bold": "173 186 199",
  "--color-prettylights-syntax-markup-changed-bg": "104 45 15",
  "--color-prettylights-syntax-markup-changed-text": "255 221 176",
  "--color-prettylights-syntax-markup-deleted-bg": "120 25 27",
  "--color-prettylights-syntax-markup-deleted-text": "255 216 211",
  "--color-prettylights-syntax-markup-heading": "49 109 202",
  "--color-prettylights-syntax-markup-ignored-bg": "37 90 178",
  "--color-prettylights-syntax-markup-ignored-text": "173 186 199",
  "--color-prettylights-syntax-markup-inserted-bg": "27 71 33",
  "--color-prettylights-syntax-markup-inserted-text": "180 241 180",
  "--color-prettylights-syntax-markup-italic": "173 186 199",
  "--color-prettylights-syntax-markup-list": "234 197 95",
  "--color-prettylights-syntax-meta-diff-range": "220 189 251",
  "--color-prettylights-syntax-storage-modifier-import": "173 186 199",
  "--color-prettylights-syntax-string": "150 208 255",
  "--color-prettylights-syntax-string-regexp": "141 219 140",
  "--color-prettylights-syntax-sublimelinter-gutter-mark": "84 93 104",
  "--color-prettylights-syntax-variable": "246 157 80",
  "--color-primer-border-active": "236 119 92",
  "--color-primer-border-contrast": "205 217 229",
  "--color-primer-canvas-backdrop": "28 33 40",
  "--color-primer-canvas-sticky": "34 39 46",
  "--color-primer-fg-disabled": "84 93 104",
  "--color-primer-shadow-focus": "0 0 0",
  "--color-primer-shadow-highlight": "0 0 0",
  "--color-primer-shadow-inset": "0 0 0",
  "--color-project-gradient-in": "45 51 59",
  "--color-project-gradient-out": "45 51 59",
  "--color-project-header-bg": "34 39 46",
  "--color-project-sidebar-bg": "45 51 59",
  "--color-scale-black": "28 33 40",
  "--color-scale-blue-0": "198 230 255",
  "--color-scale-blue-1": "150 208 255",
  "--color-scale-blue-2": "108 182 255",
  "--color-scale-blue-3": "83 155 245",
  "--color-scale-blue-4": "65 132 228",
  "--color-scale-blue-5": "49 109 202",
  "--color-scale-blue-6": "37 90 178",
  "--color-scale-blue-7": "27 75 145",
  "--color-scale-blue-8": "20 61 121",
  "--color-scale-blue-9": "15 45 92",
  "--color-scale-coral-0": "255 218 207",
  "--color-scale-coral-1": "255 185 165",
  "--color-scale-coral-2": "247 153 129",
  "--color-scale-coral-3": "236 119 92",
  "--color-scale-coral-4": "222 91 65",
  "--color-scale-coral-5": "194 68 45",
  "--color-scale-coral-6": "169 53 36",
  "--color-scale-coral-7": "141 41 27",
  "--color-scale-coral-8": "119 29 19",
  "--color-scale-coral-9": "93 16 8",
  "--color-scale-gray-0": "205 217 229",
  "--color-scale-gray-1": "173 186 199",
  "--color-scale-gray-2": "144 157 171",
  "--color-scale-gray-3": "118 131 144",
  "--color-scale-gray-4": "99 110 123",
  "--color-scale-gray-5": "84 93 104",
  "--color-scale-gray-6": "68 76 86",
  "--color-scale-gray-7": "55 62 71",
  "--color-scale-gray-8": "45 51 59",
  "--color-scale-gray-9": "34 39 46",
  "--color-scale-green-0": "180 241 180",
  "--color-scale-green-1": "141 219 140",
  "--color-scale-green-2": "107 196 109",
  "--color-scale-green-3": "87 171 90",
  "--color-scale-green-4": "70 149 74",
  "--color-scale-green-5": "52 125 57",
  "--color-scale-green-6": "43 106 48",
  "--color-scale-green-7": "36 88 41",
  "--color-scale-green-8": "27 71 33",
  "--color-scale-green-9": "17 52 23",
  "--color-scale-orange-0": "255 221 176",
  "--color-scale-orange-1": "255 188 111",
  "--color-scale-orange-2": "246 157 80",
  "--color-scale-orange-3": "224 130 61",
  "--color-scale-orange-4": "204 107 44",
  "--color-scale-orange-5": "174 86 34",
  "--color-scale-orange-6": "148 71 27",
  "--color-scale-orange-7": "127 57 19",
  "--color-scale-orange-8": "104 45 15",
  "--color-scale-orange-9": "77 33 12",
  "--color-scale-pink-0": "255 215 235",
  "--color-scale-pink-1": "255 179 216",
  "--color-scale-pink-2": "252 141 199",
  "--color-scale-pink-3": "226 117 173",
  "--color-scale-pink-4": "201 97 152",
  "--color-scale-pink-5": "174 76 130",
  "--color-scale-pink-6": "152 59 110",
  "--color-scale-pink-7": "126 50 90",
  "--color-scale-pink-8": "105 38 74",
  "--color-scale-pink-9": "85 22 57",
  "--color-scale-purple-0": "238 220 255",
  "--color-scale-purple-1": "220 189 251",
  "--color-scale-purple-2": "220 189 251",
  "--color-scale-purple-3": "176 131 240",
  "--color-scale-purple-4": "152 110 226",
  "--color-scale-purple-5": "130 86 208",
  "--color-scale-purple-6": "107 68 188",
  "--color-scale-purple-7": "89 54 162",
  "--color-scale-purple-8": "71 44 130",
  "--color-scale-purple-9": "53 33 96",
  "--color-scale-red-0": "255 216 211",
  "--color-scale-red-1": "255 184 176",
  "--color-scale-red-2": "255 147 138",
  "--color-scale-red-3": "244 112 103",
  "--color-scale-red-4": "229 83 75",
  "--color-scale-red-5": "201 60 55",
  "--color-scale-red-6": "173 46 44",
  "--color-scale-red-7": "146 35 35",
  "--color-scale-red-8": "120 25 27",
  "--color-scale-red-9": "93 15 18",
  "--color-scale-white": "205 217 229",
  "--color-scale-yellow-0": "251 224 144",
  "--color-scale-yellow-1": "234 197 95",
  "--color-scale-yellow-2": "218 170 63",
  "--color-scale-yellow-3": "198 144 38",
  "--color-scale-yellow-4": "174 124 20",
  "--color-scale-yellow-5": "150 102 0",
  "--color-scale-yellow-6": "128 84 0",
  "--color-scale-yellow-7": "108 68 0",
  "--color-scale-yellow-8": "89 54 0",
  "--color-scale-yellow-9": "69 39 0",
  "--color-search-keyword-hl": "198 144 38",
  "--color-select-menu-backdrop-border": "84 93 104",
  "--color-select-menu-tap-focus-bg": "20 61 121",
  "--color-select-menu-tap-highlight": "68 76 86",
  "--color-severe-emphasis": "174 86 34",
  "--color-severe-fg": "204 107 44",
  "--color-severe-muted": "204 107 44",
  "--color-severe-subtle": "204 107 44",
  "--color-shadow-extra-large": "0 0 0",
  "--color-shadow-large": "0 0 0",
  "--color-shadow-medium": "0 0 0",
  "--color-shadow-small": "0 0 0",
  "--color-sidenav-selected-bg": "55 62 71",
  "--color-sponsors-emphasis": "174 76 130",
  "--color-sponsors-fg": "201 97 152",
  "--color-sponsors-muted": "201 97 152",
  "--color-sponsors-subtle": "201 97 152",
  "--color-success-emphasis": "52 125 57",
  "--color-success-fg": "87 171 90",
  "--color-success-muted": "70 149 74",
  "--color-success-subtle": "70 149 74",
  "--color-timeline-badge-bg": "55 62 71",
  "--color-topic-tag-border": "0 0 0",
  "--color-underlinenav-border-hover": "99 110 123",
  "--color-underlinenav-icon": "84 93 104",
}
`;
